llmspec/LLMSpec.py

```diff
@@ -0,0 +1,38 @@
+import json
+
+from pydantic import BaseModel
+
+
+class LLMSpec(BaseModel):
```

Comment:
Never used `BaseModel`?

Reply:
It helps validate the attribute types?

Reply:
Not in the dependencies. There are some performance issues with v1*.